### PR TITLE
Combine LookupU8 and LookupU16 under Lookup(Table)

### PR DESCRIPTION
**Pull request by zemse**
### Description

Combines LookupU8 and LookupU16 under Lookup(Table).

### Issue Link

#1510

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Refactor

### CODE REVIEW

File: zkevm-circuits/src/evm_circuit/util.rs [hunk 1]
```diff
@@ -250,15 +253,15 @@ pub(crate) fn evm_cm_distribute_advice<F: Field>(
...
-        dist.add(CellType::LookupU8, advices[column_idx]);
+        dist.add(CellType::Lookup(Table::U8), advices[column_idx]);
         assert_eq!(advices[column_idx].column_type().phase(), 0);
         column_idx += 1;
     }
 
     // Mark columns used for byte lookup
     #[allow(clippy::reversed_empty_ranges)]
     for _ in 0..N_U16_LOOKUPS {
-        dist.add(CellType::LookupU16, advices[column_idx]);
```
Comment (hero78119) [inline zkevm-circuits/src/evm_circuit/util.rs:261]: Here got typo.
Test still test, suggest adding some unittest to check that

File: zkevm-circuits/src/evm_circuit/util.rs [hunk 2]
```diff
@@ -250,15 +253,15 @@ pub(crate) fn evm_cm_distribute_advice<F: Field>(
...
+        dist.add(CellType::Lookup(Table::U8), advices[column_idx]);
```
Comment (hero78119) [inline zkevm-circuits/src/evm_circuit/util.rs:264]: Here got typo.
Test still passed, suggest adding some unittest to check that

Comment (zemse) [reply zkevm-circuits/src/evm_circuit/util.rs:264]: Oh dang, I expected this mistake so I reviewed it twice locally before committing still messed it up :( Thanks for pointing!

**Review by hero78119 — APPROVED** [review]
LGTM! thanks for the contribution 🥇

**Review by ChihChengLiang — APPROVED** [review]
LGTM. Beautiful refactor. Thanks for the contribution.

**Pull request merged**